src/Microsoft.Diagnostic.TestHelpers/TestConfiguration.cs

```diff
@@ -408,7 +422,7 @@ public string DebuggeeNativeLibRoot
         }
 
         /// <summary>
-        /// The version of the Microsoft.NETCore.App package to reference when building the debuggee.
+        /// The version of the Microsoft.NETCore.App package to reference when building or running the debuggee.
```

Comment:
I would leave this property explicitly as the version of the runtime the app was built against and create a new property if you want to track what it will run against. The two versions are not required to be the same.